docs/api/io/io-config.md

```diff
@@ -2,7 +2,7 @@
 
 The driver configurations allows for customization of the OS driver layer. Options in this section pertain to hardware IO. You can find additional serial settings for `printf` in the [platform config page](configuration-platform.html). With the exception of `default-serial-baud-rate` these settings do not affect Serial objects.
 
-This is the complete list of driver configuration parameters. To view all configuration parameters, run the `--config -v` command.. Please see [the configuration system documentation](configuration.html) for details on how you may use or override these settings.
+This is the complete list of driver configuration parameters. To view all configuration parameters, run the `--config -v` command. Please see [the configuration system documentation](configuration.html) for details on how you may use or override these settings.
 
 ```
 Configuration parameters
```

Comment:
Are these config params still correct?

Reply:
That's an odd link format

Reply:
@rajkan01 are the config parameters still correct?

Reply:
@iriark01 are you asking about the command "--config -v" or the configuration specified in the documentation(configuration.html)

Reply:
The drivers output from config -v and whether it shows any drivers config params anymore (because UART serial has been deprecated)

Reply:
`UART Serial` config listed out by `--config -v` refers to the new `BufferedSerial` config option and that is expected.

Reply:
I think that requires clarifying in the docs, and perhaps worth a consideration on the code side (I've already spoken to Evelyne about the code side).